test/installer/scripts/omsadmin_systest.rb

```diff
@@ -194,11 +202,36 @@ def set_omsagent_proc_limit(val = nil, should_succeed = true)
     return output
   end
 
+  def show_omsagent_proc_limit
+    lstderrprefix='/tmp/show_omsagent_proc_limit'
+    stdout_str = `#{@omsadmin_script} -c 2>#{lstderrprefix}.stderrbuffer`
+    ret_code = $?
+    stderr_str = `cat #{lstderrprefix}.stderrbuffer`
+    `rm -f #{lstderrprefix}.stderrbuffer`
+
+    assert_equal(0, ret_code.to_i, "The command to show the user process limit was unsuccessful")
+    return stdout_str,stderr_str
+  end
+
   def unset_proc_limit
     output = `#{@omsadmin_script} -r`
     ret_code = $?
     assert_equal(0, ret_code.to_i, "The command to unset the proc limit failed")
     return output
   end
 
+  def test_show_omsagent_proc_limit_default
+    set_omsagent_proc_limit()
+    lso, lse = show_omsagent_proc_limit()
+    assert_match(/^-?\d+$/,lso,"The command to show what should be the default proc limit did not yield an integer.")
+    assert_equal(lse, "", "Unexpected error: '#{lse}")
+  end
+
+  def test_show_omsagent_proc_limit_5000
+    set_omsagent_proc_limit(5000)
+    lso, lse = show_omsagent_proc_limit()
+    assert_match(/^5000$/,lso,"Should have been 5000 process limit.  Instead #{lso}.")
```

Comment:
Ruby's assert_match will automatically print out what the expected pattern was and what it found instead; you don't have to do it yourself.